wkcuber/wkcuber/_internal/utils.py

```diff
@@ -65,6 +68,34 @@ def parse_padding(padding_str: str) -> Tuple[int, ...]:
         raise argparse.ArgumentTypeError("The padding could not be parsed") from e
 
 
+class AsciiArgumentParser(ArgumentParser):
+    """
+    This class is a lightweight wrapper around ArgumentParser and checks
+    that the shell command doesn't contain non-ascii characters since these
+    can easily disturb the argument parsing.
+
+    If non-ascii characters are found, an error is raised. This behavior
+    can be disabled by passing --allow-non-ascii on the command line.
+    """
+
+    IGNORE_FLAG = "--allow-non-ascii"
+
+    def parse_args(self, args: Optional[Sequence[str]] = None, namespace: Optional[Namespace] = None) -> Any:  # type: ignore
+        if args is None:
+            args = sys.argv[1:]
+
+        results = re.findall(r"[^\x00-\x7F]", " ".join(args))
+        if len(results) > 0:
+            if self.IGNORE_FLAG in args:
+                args = [el for el in sys.argv if el != self.IGNORE_FLAG]
+            else:
+                raise ValueError(
+                    f"The shell command contains non-ascii characters. Please remove them to avoid unintended effects. Alternatively, pass the argument {self.IGNORE_FLAG}. The special characters are: {results}"
```

Comment:
```suggestion
                    f"The shell command contains non-ascii characters. Please remove them to avoid unintended effects. If you need them, pass {self.IGNORE_FLAG} to suppress this error. The special characters are: {results}"
```